php-fpm/templates/pool.conf

```diff
@@ -75,8 +75,7 @@ daemonize = no
 ;   '/path/to/unix/socket' - to listen on a unix socket.
 ; Note: This value is mandatory.
 
-listen = /var/run/php5-fpm/{{ pool_name }}.sock
-
+listen = 127.0.0.1:9000
```

Comment:
don't hardcode port as this prevents from running more than one pool per server (i.e. see case when two separate apps are running on the same server)